support-frontend/app/controllers/Application.scala

```diff
@@ -248,7 +248,10 @@ class Application(
   }
 
   def healthcheck: Action[AnyContent] = PrivateAction {
-    Ok("healthy")
+    if (priceSummaryServiceProvider.forUser(false).getPrices(SupporterPlus, List()).isEmpty)
+      InternalServerError("no prices in catalog")
+    else
+      Ok("healthy")
```

Comment:
previously there was obviously an issue with this pricing data not being populated, hence this PR was raised https://github.com/guardian/support-frontend/pull/1752
However if there's no pricing data it's probably better not to let the instance even come up, rather than trying to alert on it?